pkgs/generate.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/bin/sh
+
```

Comment:
For each script added, can you add a short explanation at the top about what it does and when it's to be used?